exporter/influxdbexporter/go.mod

```diff
@@ -3,8 +3,8 @@ module github.com/open-telemetry/opentelemetry-collector-contrib/exporter/influx
 go 1.18
 
 require (
-	github.com/influxdata/influxdb-observability/common v0.2.27
-	github.com/influxdata/influxdb-observability/otel2influx v0.2.27
```

Comment:
Because of this, dependabot will not update these deps. So you need to not forget to move to v0.2.28 when available.